big-structure/src/destress_big_structure/analysis.py

```diff
@@ -261,7 +261,7 @@ def analyse_design(design: ampal.Assembly) -> DesignMetrics:
     try:
         ev.tag_dssp_data(design)
     except subprocess.CalledProcessError as e:
-        pass
+        print(f"Cannot compute the DSSP assignment due to a CalledProcessError:\n {e}")
```

Comment:
Should this be logging now?

Reply:
This will be left for now until we add logging to analysis.py. An issue will be made for this. However, for headless destress, the errors are caught in console.py and detailed in the logging.txt file.